enterprise/server/raft/store/store.go

```diff
@@ -69,16 +70,23 @@ const (
 	// considered to be full and eviction will kick in.
 	evictionCutoffThreshold = .90
 
-	// How often nodes wil gossip about their partition usage.
-	nodePartitionUsageGossipInterval = 15 * time.Second
+	// How often nodes wil check whether to gossip usage data if it is
+	// sufficiently different from the last broadcast.
+	nodePartitionUsageCheckInterval = 15 * time.Second
+
+	// How often nodes can go without broadcasting usage information.
+	// Usage data will be gosipped after this time if no updated were triggered
+	// based on data changes.
+	nodePartitionUsageMaxAge = 15 * time.Minute
```

Comment:
Can we s/nodes/stores/g in the comment and variable name? (just to better distinguish this from the old node usage stuff and clarify that it comes from the store)